motortorquecalculation/tests/test_alt_gain.py

```diff
@@ -0,0 +1,24 @@
+import sys
+import os.path
+import pytest
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from alt_gain import Car
+
+@pytest.fixture
+def car_default():
+    car = Car(720, 0.15, 0.0015)
+    return car
+
+def test_no_velocity(car_default):
+    assert(round(car_default.force(0, 0, 1), 1) == 10.6)
+    assert(round(car_default.energy_use(100, 0, 0, 30), 1) == 1059.5)
+    assert(round(car_default.torque_req(0, 0, 30), 1) == 56.5)
```

Comment:
This test is failing for me

Reply:
Yep, that was a calculation error on my part, sorry